pages/searchResults.page.js

```diff
@@ -0,0 +1,51 @@
+let BasePage = require('./base.page');
+let TextBox = require('../elements/textBox');
+let WebButton = require('../elements/button');
+
+let searchResultsFirstLinkLocator = by.css('.products-list > li:nth-child(1) > div.item-info > p > a');
+let itemPageHeaderLocator = by.css('div.heading > h1');
+
+class SearchResultsPage extends BasePage {
+    async waitForSearchPageToBeAvailable() {
+        await this.getPageHeaderElement().waitForElementToBeVisible(10000);
+    }
+
+    // action methods
+    async getSearchResults() {
+        await allure.createStep(`Getting search results (check that found '> 0' items)`, async () => {
+            let string = await this.getPageHeaderElement().getText();
+            let num = await string.replace(/[^0-9]/g, '');
+            let int = await parseInt(num, 10);
```

Comment:
no need `await ` in lines 17 and 18, there are no promises

Reply:
done